src/frontend/App/utils/klage.ts

```diff
@@ -1,7 +1,7 @@
 import { KlageBehandling, KlagebehandlingStatus } from '../typer/klage';
 
-export const kanOppretteBehandling = (behandlinger: KlageBehandling[]): boolean =>
-    behandlinger.every(
+export const harÅpenKlage = (behandlinger: KlageBehandling[]): boolean =>
+    !behandlinger.every(
         (behandling: KlageBehandling) =>
             behandling.status === KlagebehandlingStatus.FERDIGSTILT ||
             behandling.status === KlagebehandlingStatus.VENTER
```

Comment:
For å unngå `!` kan man bruke `some` her
```suggestion
    behandlinger.some(
        (behandling: KlageBehandling) =>
            behandling.status !== KlagebehandlingStatus.FERDIGSTILT &&
            behandling.status !== KlagebehandlingStatus.VENTER
```